cmake/libprotoc.cmake

```diff
@@ -82,6 +85,7 @@ set(libprotoc_files
   ${protobuf_source_dir}/src/google/protobuf/compiler/ruby/ruby_generator.cc
   ${protobuf_source_dir}/src/google/protobuf/compiler/subprocess.cc
   ${protobuf_source_dir}/src/google/protobuf/compiler/zip_writer.cc
+  ${js_well_known_types_embed}
```

Comment:
This list here will be overwritten when we sync the file lists and this change here will not last.